setup.py

```diff
@@ -708,6 +708,8 @@ def is_package_excluded(package: str, exclusion_list: List[str]):
     'lazy_object_proxy~=1.3',
     'lockfile>=0.12.2',
     'markdown>=2.5.2, <3.0',
+    'marshmallow<3',
```

Comment:
We don't suppport marsshmalllow yet. We have a dependency on FAB, which requires marshmallow 2.0 Support for 3.0 is WIP: https://github.com/dpgaspar/Flask-AppBuilder/pull/1382